test/source/tests/tests/gmail.ts

```diff
@@ -96,6 +96,19 @@ export const defineGmailTests = (testVariant: TestVariant, testWithNewBrowser: T
       expect(content).to.contain('STONE NEED REMAIN SLIDE DEPOSIT BRICK');
     }));
 
+    ava.default('mail.google.com[global:no.pgp] - process to flowcrypt website and decypt the message', testWithNewBrowser(async (t, browser) => {
+      const gmailPage = await browser.newPage(t, 'https://accounts.google.com/AccountChooser?service=mail&continue=https://mail.google.com/mail/');
```

Comment:
You don't actually need to log in as the recipient. Use a mock. In the sent mock email, there will be URL link to the website. Open that link with Puppeteer.

Reply:
Got it, thanks!